packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -170,6 +170,13 @@ const ComboboxField = ({
     }
   }
 
+  if (customProps.placeholder) {
+    // eslint-disable-next-line no-console
+    console.warn('[WARN] The placeholder prop has been deprecated to provide for better accessibility and a common and consistent placeholder pattern');
```

Comment:
Does this 👇  sounds better :)  ...??? 
`The placeholder prop has been deprecated and replaced with default placeholder *** Select **** for better accessibility and consistency.`

Reply:
I like it

Reply:
updated here https://github.com/cerner/terra-core/pull/3216/commits/af4a0459a6d3cf0703115e655a08c0786a263fe6